lib/models/achievement.js

```diff
@@ -31,6 +31,15 @@ module.exports = function (sequelize, DataTypes) {
     },
     media: {
       type: DataTypes.TEXT
+    },
+    icon_name: {
+      type: DataTypes.TEXT
```

Comment:
This is listed as `DataTypes.String` in the migration file, please make this match
```suggestion
      type: DataTypes.STRING
```